src/dynarec/la64/dynarec_la64_emit_math.c

```diff
@@ -592,6 +588,69 @@ void emit_sub32c(dynarec_la64_t* dyn, int ninst, rex_t rex, int s1, int64_t c, i
 }
 
 
+// emit SBB8 instruction, from s1, s2, store result in s1 using s3, s4 and s5 as scratch
+void emit_sbb8(dynarec_la64_t* dyn, int ninst, int s1, int s2, int s3, int s4, int s5)
+{
+    IFX (X_PEND) {
+        ST_B(s1, xEmu, offsetof(x64emu_t, op1));
+        ST_B(s2, xEmu, offsetof(x64emu_t, op2));
+        SET_DF(s3, d_sbb8);
+    } else IFX (X_ALL) {
+        SET_DFNONE();
+    }
+
+
+    if (la64_lbt) {
+        SBC_B(s3, s1, s2);
+        IFX (X_ALL) {
+            X64_SBC_B(s1, s2);
+        }
+
+        MV(s1, s3);
+        IFX (X_PEND)
+            ST_B(s1, xEmu, offsetof(x64emu_t, res));
+        return;
+    }
+
+
+    IFX (X_AF | X_CF | X_OF) {
+        // for later flag calculation
+        NOR(s5, xZR, s1);
+    }
+
+    SUB_W(s1, s1, s2);
+    ANDI(s3, xFlags, 1 << F_CF);
+    SUB_W(s1, s1, s3);
+    ANDI(s1, s1, 0xff);
```

Comment:
You are sure this AND is needed? that would remove a temporary carry. I'm not sure either, just asking here.